### DISCUSSION TITLE
第3节课笔记(24班)

### DISCUSSION THREAD

**Discussion by zhjunqin** [InternLM/Tutorial#356] [笔记]
第3节课笔记(24班)

**Comment by williamleo0730** [comment]
https://zhuanlan.zhihu.com/p/677985743

**Comment by kelenlv** [comment]
https://github.com/kelenlv/LLM_notes/blob/main/Day3.md

**Comment by yezhou05** [comment]
https://app.yinxiang.com/fx/c1d131a8-7791-427c-a127-067ca451b289

**Comment by ch521-od** [comment]
https://blog.csdn.net/qq_74261455/article/details/135610296?spm=1001.2014.3001.5502

**Comment by yansuoyuli** [comment]
https://blog.csdn.net/m0_58431543/article/details/135575619

**Comment by waurblove** [comment]
https://www.bilibili.com/opus/885721367103668242?spm_id_from=333.999.0.0

**Comment by Xinyi-Cheng** [comment]
https://zhuanlan.zhihu.com/p/678590926

**Comment by lindsey-chang** [comment]
https://blog.csdn.net/qq_37397652/article/details/135586700?spm=1001.2014.3001.5501

**Comment by PopoData** [comment]
https://zhuanlan.zhihu.com/p/678916028

**Comment by flh1107** [comment]
https://www.bilibili.com/read/cv29832366/

**Comment by robotzm** [comment]
https://blog.csdn.net/weixin_48319544/article/details/135722286?spm=1001.2014.3001.5501

**Comment by ljn12yyds** [comment]
https://blog.csdn.net/ljn20001229/article/details/135729103?spm=1001.2014.3001.5502
